project-react/src/pages/useApi.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import {useEffect,useState} from "react";
+import "./UseApi.css";
+export const useApi = (url) => {
+const [users,setUsers] = useState([]);
```

Comment:
what about naming it to 
const [data, setData] ? so we can use with another api ? that may not be calling users , so this way will make it more generic ?

Reply:
Yes, that is better.
Well, I will change it now, I wrote the name like this because until now I was only dealing with User, but I was going to change it later.